client/src/App.js

```diff
@@ -1,17 +1,20 @@
 import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
+import { BrowserRouter, Route, Redirect } from "react-router-dom";
 
 import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
+import Header from "./components/Header";
+import Home from "./pages/Home";
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Header/>
+        <Route exact path={["/", "/home", "/home/:page"]} render={(props) => <Home {...props} />} />
```

Comment:
@sina-jamshidi I added in react router for the tabs, and the url changes when I click on them. Unfortunately, I was not able to get it working with the browser back button. I think we can leave as it is for now?

Reply:
Okay let's leave it for now